src/pages/PoolsOverview.tsx

```diff
@@ -1,5 +1,21 @@
 import { Layout } from "@stellar/design-system";
+import { Avatar } from "components/Avatar";
 
 export const PoolsOverview = () => (
-  <Layout.Inset>Pools overview page</Layout.Inset>
+  <Layout.Inset>
+    Pools overview page
+    <div style={{ marginTop: 20 }}>
+      <Avatar
+        source={[
+          {
+            asset:
+              "USDC:GA5ZSEJYB37JRC5AVCIA5MOP4RHTM335X2KGX3IHOJAPP5RE34K4KZVN",
+          },
+          {
+            asset: "native",
```

Comment:
these asset strings match what we get from Horizon:

https://horizon-testnet.stellar.org/liquidity_pools/2389378a6156eedfac66daa000d24c926431c3e667b9f754771964f27a6da6ab/